### PR TITLE
Rename regions to panels and reporter groups to cohorts

### PR DESCRIPTION
**Pull request by dpoirier**
See https://github.com/rapidpro/tracpro/issues/3
Name changes:
Regions -> Panels
Reporter Groups -> Cohorts

### CODE REVIEW

File: tracpro/polls/views.py [hunk 1]
```diff
@@ -600,7 +600,10 @@ def render_to_response(self, context, **response_kwargs):
                 questions = self.derive_questions().values()
 
                 resp_headers = ['Date']
-                contact_headers = ['Name', 'URN', 'Region', 'Group']
+                # FIXME: Should we change the column name "Region" to "Panel" and possibly break
+                # existing integrations that work by exporting .CSVs?
+                # Or leave it as "Region", and confuse people trying to create new integrations?
```
Comment (emullaney) [inline tracpro/polls/views.py]: Dan, can we hipchat about this one whenever you have time? No rush.

Comment (emullaney) [inline tracpro/polls/views.py]: The short answer is that I'm not concerned about existing integrations (I doubt that there are any to worry about). I would rather not confuse people, I would suggest we use the new name here.

**Comment by emullaney** [comment]
Caught one missing change on this page: http://caktus.localhost:8000/pollrun/participation/10/?group-by=region
image

Wording on the page says "Region" on the column header. Not concerned about the url.

**Comment by emullaney** [comment]
"Region" should be "Panel" here on this column header as well, which I think I mentioned already in regards to a FIXME comment: http://caktus.localhost:8000/response/by_pollrun/13/

**Comment by emullaney** [comment]
@dpoirier this is ready for some suggested fixes. I've reviewed it in it's entirety. Looks pretty great so far.

**Comment by dpoirier** [comment]
@emullaney This has been updated and is ready for re-review.

**Comment by emullaney** [comment]
:sunflower:

**Pull request merged**